README.md

```diff
@@ -34,5 +34,7 @@ The main Labels are as follows with the corresponding meaning:
 * [Status: In Progress](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20In%20Progress): A user have mentioned that they have started to work on the request, and/or a new maintainer have been added to the existing package.
 * [Status: Review](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20Review): A package for the current request have been submitted to chocolatey.org, and is currently awaiting a review from the chocolatey community moderators.
 * [Status: Published](https://github.com/chocolatey/chocolatey-package-requests/labels/Status%3A%20Published): The package for the current request have been approved on the chocolatey community repository, and is now available for everyone to use.
-* [Blocked Upstream](https://github.com/chocolatey/chocolatey-package-requests/labels/Blocked%20Upstream): There are known issues that need to be resolved before a package can be created for the current request. (This can be everything from needing to embedd a package for a software that does not grant any distribution rights, download is blocked behind a login form, additional headers needed to be sent with the download request, and more...).
-* [Duplicate](https://github.com/chocolatey/chocolatey-package-requests/labels/Duplicate): The current request is a duplicate of a previous open request.
+* [Status: Blocked Upstream](https://github.com/chocolatey/chocolatey-package-requests/labels/Blocked%20Upstream): There are known issues that need to be resolved before a package can be created for the current request. (This can be everything from needing to embedd a package for a software that does not grant any distribution rights, download is blocked behind a login form, additional headers needed to be sent with the download request, and more...).
```

Comment:
`Blocked Upstream` and `Duplicate` isn't really a status though IMO.
(Well maybe `Duplicate`).
Even if it's blocked upstream, it doesn't mean it can't be done.

The same thing would be for the Other labels you documented IMO.

Reply:
If it can be done it shouldn't be labelled as `Blocked Upstream`. I only see that label being used if the package is currently blocked and when things start working again then that label will be changed to `In Progress` or another appropriate label. If it's cleared it can't be progressed then it should be clocked with the `Blocked Upstream` label.

Reply:
I disagree (however, the label should probably be renamed).

The intention was to notify would-be maintainer that additional work is required to make a package.
Say, for instance the package needs to be embedded because a remote url can't be used (for instance the download requires registration, feedback or more).

Or the url to uses specific hashes appended to the url to make them available to work, or a different useragent, referer or something completely else.

Of course, it could be a status as well, in cases where there are issues that currently can't be resolved

Reply:
@AdmiringWorm Can we get some consensus around this to allow this to progress? And if we agree to rename this what it should be named as?